## ~/.pdbrc or in `pwd`/.pdbrc
## After getting an error start the python program like this:
## bash $ python -m pdb my-failing-program.py
## this will start debugger and the following commands will be run...
##
## Alternatively, you can run pdb.pm() in the python repl to start a postmortem debugger!
list
where

## From https://docs.python.org/3/library/pdb.html#pdbcommand-alias
alias pi for k in %1.__dict__.keys(): print(f"%1.{k} = {%1.__dict__[k]}")
alias ps pi self
